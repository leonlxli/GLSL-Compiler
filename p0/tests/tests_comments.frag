/* comment all of this
******** ///// / this line should not show
 dasff +1 -1 
 */   	true

a//FALSE should not show

sentence		has 		 	spaces 		and tabs 	

False

/*

ldsakjflkajdf
kladsjflkajf
daslfjalkfjas
/////*