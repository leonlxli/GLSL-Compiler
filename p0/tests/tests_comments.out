true         line 4 cols 15-18 is T_BoolConstant (value = true)
a            line 6 cols 1-1 is T_Identifier 
sentence     line 8 cols 1-8 is T_Identifier 
has          line 8 cols 25-27 is T_Identifier 
spaces       line 8 cols 54-59 is T_Identifier 
and          line 8 cols 77-79 is T_Identifier 
tabs         line 8 cols 81-84 is T_Identifier 
False        line 10 cols 1-5 is T_Identifier 

*** Error.
*** Input ends with unterminated comment

